pkg/config/config_template.yaml

```diff
@@ -373,6 +373,11 @@ api_key:
   #
```

Comment:
Why did you leave out `apm_config.additional_endpoints` and `apm_dd_url`?

Reply:
As stated in the PR description what could be the usage for `apm_config.additional_endpoints` ? I'm going to add the `apm_dd_url` for proxy usage though, that's a miss on my end.